django/db/models/sql/compiler.py

```diff
@@ -1115,7 +1117,7 @@ def as_sql(self):
             raise EmptyResultSet
         sql, params = [], []
         for annotation in self.query.annotation_select.values():
-            agg_sql, agg_params = self.compile(annotation)
+            agg_sql, agg_params = self.compile(annotation, select_format=True)
```

Comment:
I suggested that because `annotation_select` now contains expressions other than aggregates. But I didn't notice the method was in the AggregateCompiler, so it probably is just aggregates in that dict. Not worth worrying about I guess.